input/docs/getting-started/running-cake-scripts.md

```diff
@@ -0,0 +1,61 @@
+Order: 10
+---
+
+There are different runners available for running Cake scripts.
+
+# .NET Core Tool
+
+:::{.alert .alert-info}
+This is the recommended way to run Cake Scripts.
+:::
+
+The [Cake.Tool](https://www.nuget.org/packages/Cake.Tool) NuGet package, is a .NET Core global tool compiled for .NET Core 2.1 or newer.
+
+:::{.alert .alert-info}
+The following instructions require .NET Core 3.0 or newer.
+See [How to manage .NET Core tools](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools) for details and other options.
+:::
+
+Make sure to have a tool manifest available in your repository or create one using the following command:
+
+```shell
+dotnet new tool-manifest
+```
+
+Install Cake as a local tool using the `dotnet tool` command:
+
+```shell
+dotnet tool install Cake.Tool --version x.y.z
+```
+
+Once installed, you can launch Cake from anywhere using the .NET CLI:
+
+```shell
+dotnet cake
+```
+
+:::{.alert .alert-info}
+By convention this will execute the build script named `build.cake`.
+You can override this behavior by additionally passing the name of the build script.
+:::
+
+# Cake runner for .NET Framework
+
+The [Cake](https://www.nuget.org/packages/Cake) NuGet package is a runner requiring [.NET Framework 4.6.1](https://www.microsoft.com/net/download/dotnet-framework/net461)
+or newer on Windows and Mono `5.12.0` or newer on Mac or Linux.
+
+:::{.alert .alert-warning}
+It is suggested to use [.NET Framework 4.7.2](https://www.microsoft.com/net/download/dotnet-framework/net472) or newer to run build scripts
+which are using addins targeting .NET Standard 2.0 only.
+:::
+
+The runner is mainly for backwards compatibility where scripts or addins are used which require .NET Framework.
+In all other cases it is recommended to use [.NET Core Tool](#net-core-tool).
```

Comment:
Is it worth mentioning "how" you can install this?  I am thinking either through homebrew, or Chocolatey, or through using a bootstapper script?

Reply:
I think #931 should solve this

Reply:
Fair.  Once that is in place, I think we should link through to that from here.